_posts/2020-05-15-Performance-testing-in-shallow-waters.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Performance testing in shallow waters
+banner : images/banner-pulse-articles5.jpg
+date:  2020-05-15
+categories:
+- wnc
+- media
+thumbnail: images/pulse-thumbnail-deployment.jpg
+---
+
+Acoustic wireless communications have always been very challenging in shallow coastal waters as compared to the deep ocean. This is due to several factors such as multi-path, Doppler due to strong currents, varying bathymetry, etc. It gets even more challenging when you have to share the propagation channel with several marine vessels constantly injecting bubbles into the communication channel.
+
+This is why, we at Subnero, constantly test our underwater acoustic modems at these challenging conditions, to push the limits of what is possible.
+
+During February 2020, we conducted a set of experiments at the [Singapore Strait](https://en.wikipedia.org/wiki/Singapore_Strait), to test the communication range between two of our modems in shallow waters. The goal of the test was to establish a maximum communication range at a given data rate with a more than 70% packet success ratio.
```

Comment:
I think that it should be "in" the Singapore Strait instead of "at" the Singapore Strait, however, I can't find anything to concur or disagree with that.

Reply:
For now, I am leaving it with `at`.